src/cc2olx/tools/README.rst

```diff
@@ -6,7 +6,7 @@ Tools
 Video Upload Tool
 -----------------
 
-The video upload tool uploads video files to edX's video encoding pipeline.
+The video upload tool uploads video files and associated transcripts to edX's video encoding pipeline.
```

Comment:
@kaizoku can we also include documentation for the video download tool? It can be added to this file, but maybe under another section

Reply:
Yes, I added a section below describing the options to the video download tool, its expected input and output, and how it can be used with the video upload tool. Does this extra section look good?

Reply:
That looks great, thanks for adding!